pkg/models/query.go

```diff
@@ -83,7 +84,7 @@ type TwinMakerListEntitiesFilter struct {
 
 func (f *TwinMakerPropertyFilter) ToTwinMakerFilter() *iottwinmaker.PropertyFilter {
 	filter := &iottwinmaker.PropertyFilter{
-		PropertyName: &f.Name,
+		PropertyName: aws.String(f.Name),
```

Comment:
this appears to fix #130
<img width="1373" alt="image" src="https://user-images.githubusercontent.com/360020/201834001-e17ebfd6-eae7-4285-a729-beb5f1d5e219.png">